client/src/main/java/io/quarkus/vault/client/http/jdk/JDKVaultHttpClient.java

```diff
@@ -0,0 +1,65 @@
+package io.quarkus.vault.client.http.jdk;
+
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpRequest.BodyPublishers;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeoutException;
+
+import io.quarkus.vault.client.common.VaultRequest;
+import io.quarkus.vault.client.common.VaultResponse;
+import io.quarkus.vault.client.http.VaultHttpClient;
+import io.smallrye.mutiny.Uni;
+
+public class JDKVaultHttpClient extends VaultHttpClient {
```

Comment:
I was wondering if there was a need to make it in a separate module, fo sake of symmetry with the vertx client. but I guess technically there is no point.

Reply:
Originally I had thought of this the same way but after completing the work I realized it's a single, _very_ small class and it adds zero dependencies.